README.md

```diff
@@ -1686,6 +1686,29 @@ You will have to invoke this as a composite flow:
 cargo make workspace-task --no-workspace
 ```
 
+<a name="usage-workspace-support-include-members"></a>
+#### Include Specific Members
```

Comment:
great. can you also add a link to this in the nav.md?

Reply:
Done.